app/packs/src/decidim/participatory_documents/admin/manage_bulk_suggestions.js

```
@@ -131,3 +135,18 @@ window.addEventListener("load", () => {
     })
   }
 });
+
+document.addEventListener("turbo:load", () => {
+  const evaluatorMultiselectContainers = document.querySelectorAll(
+    ".js-evaluator-multiselect"
+  );
+
+  evaluatorMultiselectContainers.forEach((container) => {
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)